x/slashing/app_test.go

```diff
@@ -156,7 +158,7 @@ func TestSlashingMsgs(t *testing.T) {
 	require.Equal(t, sdk.ValAddress(addr1), validator.OperatorAddress)
 	require.Equal(t, sdk.Bonded, validator.Status)
 	require.True(sdk.IntEq(t, bondTokens, validator.BondedTokens()))
-	unjailMsg := MsgUnjail{ValidatorAddr: sdk.ValAddress(validator.ConsPubKey.Address())}
+	unjailMsg := MsgUnjail{ValidatorAddr: validator.OperatorAddress}
```

Comment:
Why did this need to change?

Reply:
The address that is signing the unjail message should be the 'operating address' of the validator, which is not related to its consensus public key (now bls).